src/components/List/ListItem/ListItem.tsx

```diff
@@ -44,6 +44,40 @@ const ListItem = React.forwardRef<HTMLDivElement, Props>(
         handleOptionClick(content as never);
       }
     };
+    const renderContent = (content: ListItemType) => {
+      if (
```

Comment:
Since this is a pretty weird and complicated check right here, could you add a few comments explaining it , so we don't miss something in the future?

Reply:
i could move it to a const but current typescript doesnt support it only on 4.4 and after